src/main/ts/components/Editor.tsx

```diff
@@ -7,25 +7,27 @@
  */
 
 import * as React from 'react';
-import { EventHandler, IEvents } from '../Events';
+import { IEvents } from '../Events';
 import { ScriptLoader } from '../ScriptLoader';
 import { getTinymce } from '../TinyMCE';
 import { bindHandlers, isFunction, isTextarea, mergePlugins, uuid } from '../Utils';
 import { EditorPropTypes, IEditorPropTypes } from './EditorPropTypes';
+import { Editor as TinyMCEEditor, EditorEvent, Events, RawEditorSettings } from 'tinymce';
+
 
 export interface IProps {
   apiKey: string;
   id: string;
   inline: boolean;
   initialValue: string;
-  onEditorChange: EventHandler<any>;
+  onEditorChange: (a: string, editor: TinyMCEEditor) => void;
   value: string;
-  init: Record<string, any>;
+  init: RawEditorSettings & { selector?: undefined, target?: undefined };
```

Comment:
Maybe add a type alias for this. There's another reference to it below.

Reply:
Is this type intended to represent that "selector" and "target" cannot be set? If so, would the `Omit` type be useful?

Reply:
I tried omit first, it did something really weird and essentially became 'any'. I suspect it was something to do with our catch-all parameters. This method still works well to deny setting those parameters.

Reply:
I have changed the second reference to use `IProps['init']` instead.